api/src/main/resources/notebooks.yaml

```diff
@@ -1426,9 +1418,14 @@ definitions:
       defaultClientId:
         type: string
         description: The default Google Client ID.
-      jupyterDockerImage:
```

Comment:
Also deprecated in the Leo yaml

`toolDockerImage` and below: copy/paste update from Leo swagger yaml